packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -120,6 +121,15 @@ function webpackConfigSingleSpa(opts) {
         },
       },
       allowedHosts: "all",
+      hot: outputSystemJS,
+      setupMiddlewares: (middlewares, devServer) => {
+        const htmlDevServer = readFileSync(
+          path.join(__dirname, "index.html"),
+          "utf8"
+        );
+        devServer.app.get("/", (_, res) => res.send(htmlDevServer));
```

Comment:
I like this implementation approach - avoid outputting the HTML file when building.